src/Lock/BasicLockInformationProvider.php

```diff
@@ -22,8 +22,8 @@ public function getLockInformation()
         $hostname = gethostname();
 
         $params = array();
-        $params[] = $pid;
-        $params[] = $hostname;
+        $params['pid'] = $pid;
```

Comment:
I had to give these keys because now that there can be different information providers the order, length, and contents of the lock values will vary. The predis lock adds an 'expiration' key to this so the structure needs to be known.
Plus it just makes sense to give these keys now that the data can be anything a provider wants to use.